### PR TITLE
Various type hints

### PR DESCRIPTION
**Pull request by schopin-pro**
Here is a grab bag of type hints added while working on #295.

### CODE REVIEW

**Comment by schopin-pro** [comment]
The `crashid` failure can be ignored for this PR as it is handled in #295

File: apport/sandboxutils.py [hunk 1]
```diff
@@ -120,16 +120,17 @@ def needed_runtime_packages(report, pkgmap_cache_dir, pkg_versions, verbose=Fals
     return [(p, pkg_versions.get(p)) for p in pkgs]
 
 
+# pylint: disable=too-many-arguments
```
Comment (bdrung) [inline apport/sandboxutils.py:123]:
```suggestion
# pylint: disable-next=too-many-arguments
```

**Review by bdrung — COMMENTED** [review]
Except for one small comments, this change looks good.

**Review by bdrung — APPROVED** [review]

**Comment by bdrung** [comment]
Thanks for splitting the type hint changes.

**Pull request merged**